python/tests/test_sql.py

```diff
@@ -15,280 +15,177 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import unittest
-import tempfile
-import datetime
-import os.path
-import shutil
+import numpy as np
+import pyarrow as pa
+import pytest
+from datafusion import ExecutionContext
 
-import numpy
-import pyarrow
-import datafusion
+from . import generic as helpers
 
-# used to write parquet files
-import pyarrow.parquet
 
-from tests.generic import *
+@pytest.fixture
+def ctx():
+    return ExecutionContext()
 
 
-class TestCase(unittest.TestCase):
-    def setUp(self):
-        # Create a temporary directory
-        self.test_dir = tempfile.mkdtemp()
-        numpy.random.seed(1)
+def test_no_table(ctx):
+    # TODO(kszucs): should raise a DataFusionError instead of plain Exeption
```

Comment:
Should we move this to an issue instead of a `TODO` with your name?

Reply:
https://github.com/apache/arrow-datafusion/issues/494